http-gateway/web/src/containers/Devices/Detail/DevicesDetailsPage/Tabs/Tab1.tsx

```diff
@@ -110,12 +112,42 @@ const Tab1: FC<Props> = (props) => {
                                 <div>-</div>
                             ),
                         },
+                        /*
```

Comment:
Is this just for demonstration ?

Reply:
Yes I want to use it for demonstration.  But If you have time could you please move it in way that we can merge it to main branch ? The logic will be more complex because there are also error states (red cloud image)

https://github.com/openconnectivityfoundation/core-extensions/blob/master/swagger2.0/oic.r.softwareupdate.swagger.json